From e36d6d831170e718caf936356fd87d7b7f42dc6b Mon Sep 17 00:00:00 2001
From: Matt Porter <mporter@linaro.org>
Date: Wed, 5 Feb 2014 14:01:40 -0500
Subject: [PATCH 5/5] pwm: enable TI PWMSS if the IIO tiecap driver is selected

The IIO TI ECAP driver depends on the TI PWMSS management
driver in this subsystem. Enable PWMSS when the IIO TI ECAP
driver is selected.

(pulling in this driver for GSoC anemometer project,
 thanks matt!)

Signed-off-by: Matt Porter <mporter@linaro.org>
Signed-off-by: Stephen Arnold <sarnold@vctlabs.com>
---
 drivers/pwm/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pwm/Kconfig b/drivers/pwm/Kconfig
index 4d42f348df82..87d626da67e4 100644
--- a/drivers/pwm/Kconfig
+++ b/drivers/pwm/Kconfig
@@ -408,7 +408,7 @@ config  PWM_TIEHRPWM

 config  PWM_TIPWMSS
	bool
-	default y if (ARCH_OMAP2PLUS) && (PWM_TIECAP || PWM_TIEHRPWM)
+	default y if (ARCH_OMAP2PLUS) && (IIO_TIECAP || PWM_TIECAP || PWM_TIEHRPWM)
	help
	  PWM Subsystem driver support for AM33xx SOC.

--
2.12.2
